ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1578,6 +1578,35 @@ literals:
 - valve_status:
   - OPEN
   - CLOSED
+
+- failed_panel_alarm:
+  - ACTIVE
+  - INACTIVE
+- low_smoke_alarm: # for aspirating type smoke detectors
+  - ACTIVE
+  - INACTIVE
+- medium_smoke_alarm: # for aspirating type smoke detectors
+  - ACTIVE
+  - INACTIVE
+- particle_concentration_alarm:
+  - ACTIVE
+  - INACTIVE
+- low_battery_alarm:
+  - ACTIVE
+  - INACTIVE
+- load_bypass_status:
```

Comment:
please use existing `bypass_status` field

Reply:
Hi ma'am, for the UPS device, it would seem ambiguous to just use bypass_status as it could mean 2 things: load is bypassed  or battery is bypassed. so I used load_bypass to be much more specific. I hope this is ok.